app/views/product_drives/show.html.erb

```diff
@@ -100,3 +100,48 @@
     </div>
   </div>
 </section>
+
+<section class="content">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        <!-- Default box -->
+        <div class="card">
+          <div class="card-body p-0">
+            <table class="table">
+              <thead>
+              <tr>
+                <th>Business Name</th>
+                <th>Contact Name</th>
+                <th>Phone</th>
+                <th>Email</th>
+                <th>Address</th>
+                <th>Items for this Drive</th>
+                <th>Total Items</th>
```

Comment:
@cielf, do you think we could make it more clear that this is for all-time? (Maybe just something like
```suggestion
                <th>Total Items (for all drives)</th>
```

Reply:
Yeah, let's go with that -- we might refine it later.